proto/elys/tradeshield/tx.proto

```diff
@@ -19,6 +19,7 @@ service Msg {
   rpc DeletePendingPerpetualOrder (MsgDeletePendingPerpetualOrder) returns (MsgDeletePendingPerpetualOrderResponse);
   rpc UpdateParams                (MsgUpdateParams               ) returns (MsgUpdateParamsResponse               );
   rpc ExecuteOrders               (MsgExecuteOrders              ) returns (MsgExecuteOrdersResponse              );
+  rpc CancelSpotOrders            (MsgCancelSpotOrders           ) returns (MsgCancelSpotOrdersResponse           );
```

Comment:
@cryptokage1996 can we remove `DeletePendingSpotOrder` as it will introduce redundancy with this new message.